intermediate-exercises/src/components/02-StopWatch.js

```diff
@@ -1,26 +1,38 @@
 import React, { Component } from 'react';
 
-/*
-* Exercise 2:
-*
-*  Create a `StopWatch` component that has a Start/Stop button and a Clear
-*  button.
-*
-*  Pressing Start will start a timer and the lapsed time in
-*  milliseconds should be displayed above the buttons.
-*
-*  Once started the Start button should change to Stop. Clicking Stop
-*  will stop the timer but lapsed time will be preserved.
-*
-*  Clicking Start again will resume the timer from where it left off.
-*
-*  Clicking Clear will stop the timer if it's running and reset the lapsed time to 0.
-*/
-
 class StopWatch extends Component {
+  state = {
+    stopWatchStart: false,
+    elapsedTime: 0
+  };
+
+  toggleTimer = () => {
+    this.setState({ stopWatchStart: this.state.stopWatchStart === false });
```

Comment:
My initial assumption of toggle is something like this - 
```js
this.setState( prevState => ({
  stopWatchStart: !stopWatchStart
}));
```

Right now, this seems to be asserting that `stopWatchStart` is false.